migrations/Version202212131248212260_taoQtiTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiTest\scripts\install\RegisterQtiPackageExporter;
+
+final class Version202212131248212260_taoQtiTest extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Patch QtiTestExporter config by proper test exporter class';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->runAction(new RegisterQtiPackageExporter);
```

Comment:
We need also make sure that the change is applied to `config/taoQtiTest/TestModel.conf.php`.  Checking this file I cannot see the change there https://github.com/oat-sa/extension-tao-testqti/blob/master/scripts/install/RegisterQtiPackageExporter.php#L39

Reply:
Because it is already fixed, see [which class it uses](https://github.com/oat-sa/extension-tao-testqti/blob/master/scripts/install/RegisterQtiPackageExporter.php#L30)

Reply:
I would not touch Updater at all. It does not use anymore but keeps the historical data.

Reply:
We should not keep deleted classes in the code base. For historical data we have git hub. We already removed this class from the code 29 days ago, so it makes sense to remove all its usage.